packages/editor/src/components/parts/task/options/TaskListPart.tsx

```diff
@@ -10,7 +11,12 @@ const TaskListPart = () => {
     <PathCollapsible label='Options' defaultOpen={task.skipTasklist} path='delay'>
       <Checkbox label='Skip Tasklist' value={task.skipTasklist} onChange={change => update('skipTasklist', change)} />
       <ValidationFieldset label='Delay' {...delayFieldset.labelProps}>
-        <ScriptInput value={task.delay} onChange={change => update('delay', change)} {...delayFieldset.inputProps} type='Duration' />
+        <ScriptInput
+          value={task.delay}
+          onChange={change => update('delay', change)}
+          {...delayFieldset.inputProps}
+          type={IVY_SCRIPT_TYPES.DURATION}
```

Comment:
we did just the same in the old SWT world. So I don't see this currently as a big problem. So I think it's ok to keep it as is ... and making it easy to change if required later on ... by using these constants :+1: 
![duration-swt](https://github.com/axonivy/inscription-client/assets/15085693/24e75484-c991-47ac-bfe4-523d1b27f5eb)